pkg/lib/web/static_assets.go

```diff
@@ -31,6 +32,30 @@ var AuthgearCSS = resource.RegisterResource(CSSDescriptor{
 	Path: StaticAssetResourcePrefix + "authgear.css",
 })
 
+var IconsCSS = resource.RegisterResource(CSSDescriptor{
```

Comment:
Can we add a comment here to document the version of tabler-icon we have embedded? It is easier than to `git blame` and then see the version in the commit message.